persistence/sql/migrations/20211019000001000000_merge_authentication_request_tables.sqlite.up.sql

```diff
@@ -0,0 +1,286 @@
+CREATE TABLE hydra_flow
+(
+    challenge                 VARCHAR(40)  NOT NULL PRIMARY KEY,
+    requested_scope           TEXT         NOT NULL,
+    verifier                  VARCHAR(40)  NOT NULL,
+    csrf                      VARCHAR(40)  NOT NULL,
+    subject                   VARCHAR(255) NOT NULL,
+    request_url               TEXT         NOT NULL,
+    skip                      INTEGER      NOT NULL,
+    client_id                 VARCHAR(255) NOT NULL REFERENCES hydra_client (id) ON DELETE CASCADE,
+    requested_at              TIMESTAMP    NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    login_initialized_at      TIMESTAMP    NULL,
+    oidc_context              TEXT         NOT NULL,
+    login_session_id          VARCHAR(40)  NULL REFERENCES hydra_oauth2_authentication_session (id) ON DELETE CASCADE DEFAULT '',
+    requested_at_audience     TEXT         NULL DEFAULT '',
+
+    state                     INTEGER      NOT NULL DEFAULT 0,
+
+    remember                  INTEGER      NULL,
+    remember_for              INTEGER      NULL,
+    error                     TEXT         NULL,
+    acr                       TEXT         NULL,
+    login_authenticated_at    TIMESTAMP    NULL,
+    was_used                  INTEGER      NULL,
+    forced_subject_identifier VARCHAR(255) NULL DEFAULT '',
+    context                   TEXT         NULL DEFAULT '{}',
+    amr                       TEXT         NULL DEFAULT '',
+
+    consent_challenge_id      VARCHAR(40)  NULL,
+    consent_skip              INTEGER      NOT NULL,
+    consent_verifier          VARCHAR(40)  NOT NULL,
+    consent_csrf              VARCHAR(40)  NOT NULL,
+
+    ch_granted_scope         TEXT        NOT NULL,
+    ch_granted_at_audience   TEXT        NULL DEFAULT '',
+    ch_remember              INTEGER     NOT NULL,
+    ch_remember_for          INTEGER     NOT NULL,
+    ch_handled_at            TIMESTAMP   NULL,
+    ch_was_used              INTEGER     NOT NULL,
+    ch_error                 TEXT        NOT NULL,
+    ch_session_id_token      TEXT        NOT NULL,
+    ch_session_access_token  TEXT        NOT NULL,
```

Comment:
Same as above - we have `consent_`, `` (empty), `ch_` prefixes. I would prefer to use a common language to better understand what field refers to what

Reply:
Agree. How about this? The fields in the Flow struct would be renamed accordingly, including WasHandled -> WasUsed.
```
challenge                ->  login_challenge
requested_scope
verifier                 ->  login_verifier
csrf                     ->  login_csrf
subject
request_url
skip                     ->  login_skip
client_id
requested_at
login_initialized_at
oidc_context
login_session_id
requested_at_audience
state
remember                 ->  login_remember
remember_for             ->  login_remember_for
error                    ->  login_error
acr
login_authenticated_at
was_used                 ->  login_was_used
forced_subject_identifier
context
amr
consent_challenge_id
consent_skip
consent_verifier
consent_csrf
ch_granted_scope         ->  granted_scope
ch_granted_at_audience   ->  granted_at_audience
ch_remember              ->  consent_remember
ch_remember_for          ->  consent_remember_for
ch_handled_at            ->  consent_handled_at
ch_was_used              ->  consent_was_used
ch_error                 ->  consent_error
ch_session_id_token      ->  session_id_token
ch_session_access_token  ->  session_access_token

```